.github/workflows/stale.yml

```diff
@@ -0,0 +1,38 @@
+on:
+  schedule:
+    - cron: '0 * * * *'
+
+jobs:
+  Prune Stale:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Prune Stale
+      uses: actions/stale@v3.0.14
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        # Different amounts of days for issues/PRs are not currently supported but there is a PR
+        # open for it: https://github.com/actions/stale/issues/214
+        days-before-stale: 30
```

Comment:
very belated question, I think if we split this into stale_prs.yaml and stale_issues.yaml I think we could reinstate the tighter stale tagging on PRs.  Think it's worth it?  If so I'm happy to give it a try.